packages/webpack-extraction-plugin/__fixtures__/webpack/style-buckets/code.ts

```diff
@@ -0,0 +1,20 @@
+import { __styles } from '@griffel/react';
+
+const styles = __styles(
+  {},
+  // Classes in this test are intentionally not realistic to simplify snapshots
+  {
+    d: ['.color-red { color: red; }', '.animation-name { animation-name: foo; }'],
+    l: ['.color-orange:link { color: orange; }'],
+    v: ['.color-purple:visited { color: purple; }'],
+    w: ['.color-pink:focus-within { color: pink; }'],
+    f: ['.color-blue:focus { color: blue; }'],
+    i: ['.color-light-blue:focus-visible { color: salmon; }'],
+    h: ['.color-yellow:hover { color: yellow; }'],
+    a: ['.color-black:active { color: black; }'],
+    k: ['@keyframes foo { from{ transform:rotate(0deg); } to { transform:rotate(360deg); } }'],
+    t: ['@media (forced-colors: active) { .color-magenta { color:magenta; } }'],
```

Comment:
```suggestion
    m: ['@media (forced-colors: active) { .color-magenta { color:magenta; } }'],
```

shouldn't this be bucket `m` ?

Reply:
Yeah, it should. I also discovered that meta was not handled properly, see 123c2e3